cmd/util.go

```diff
@@ -290,13 +295,34 @@ Alternatively, you can specify the version with --version
 `
 
 func getKubeVersion() (*KubeVersion, error) {
+	kubeConfig, err := rest.InClusterConfig()
+	if err != nil {
+		glog.V(3).Infof("Error fetching cluster config: %s", err)
+	}
+	isRKE := false
+	if err == nil && kubeConfig != nil {
+		k8sClient, err := kubernetes.NewForConfig(kubeConfig)
+		if err != nil {
+			glog.V(3).Infof("Failed to fetch k8sClient object from kube config : %s", err)
+		}
+		if err == nil {
+			isRKE, err = providers.IsRKE(context.Background(), k8sClient)
```

Comment:
Can't we determine if the cluster is an RKE cluster based on the version of the apiserver?
If that's the case, I personally suggest implementing this simple `IsRKE` function in the kube-bench project, rather than introducing a dependency on https://github.com/rancher/kubernetes-provider-detector .